memor/session.py

```diff
@@ -378,12 +377,25 @@ def from_json(self, json_object: Union[str, Dict[str, Any]]) -> None:
                     message_obj = Response()
                 message_obj.from_json(message)
                 messages.append(message_obj)
+            memor_version = loaded_obj["memor_version"]
+            date_created = datetime.datetime.strptime(loaded_obj["date_created"], DATE_TIME_FORMAT)
+            date_modified = datetime.datetime.strptime(loaded_obj["date_modified"], DATE_TIME_FORMAT)
         except Exception:
             raise MemorValidationError(INVALID_SESSION_STRUCTURE_MESSAGE)
-        messages = messages
-        memor_version = loaded_obj["memor_version"]
-        date_created = datetime.datetime.strptime(loaded_obj["date_created"], DATE_TIME_FORMAT)
-        date_modified = datetime.datetime.strptime(loaded_obj["date_modified"], DATE_TIME_FORMAT)
+        _validate_string(title, "title")
+        _validate_pos_int(render_counter, "render_counter")
+        _validate_status(messages_status, messages)
+        _validate_string(memor_version, "memor_version")
+        return title, render_counter, messages, messages_status, memor_version, date_created, date_modified
```

Comment:
The `Session` class has many parameters, so returning them in this format is not a good approach. I suggest returning a dictionary instead.

Reply:
fixed in [9523075](https://github.com/openscilab/memor/pull/120/commits/9523075ef2ebd4a782024e43d9cd231834bd51a5)